catalyst_voices/lib/widgets/menu/voices_node_menu.dart

```diff
@@ -5,7 +5,7 @@ import 'package:collection/collection.dart';
 import 'package:equatable/equatable.dart';
 import 'package:flutter/material.dart';
 
-final class VoicesNodeMenuItem extends Equatable {
+base class VoicesNodeMenuItem extends Equatable {
   const VoicesNodeMenuItem({
```

Comment:
For this class make sure to put class fields above constructor, not below

Reply:
It should be `final`. I was trying other approach but i didn't like it after all.
Will come back to `final`.